src/xrpld/app/misc/WasmHostFuncImpl.cpp

```diff
@@ -49,28 +49,43 @@ WasmHostFunctionsImpl::getParentLedgerHash()
 }
 
 Expected<int32_t, HostFunctionError>
-WasmHostFunctionsImpl::cacheLedgerObj(uint256 const& objId, int32_t cacheIdx)
+WasmHostFunctionsImpl::normalizeCacheIndex(int32_t normalizedIdx)
```

Comment:
This function accept parameter which is just index, or rawIndex,  normalized index will be returned by this function